packages/odyssey-design-tokens/src/font.json

```diff
@@ -3,11 +3,11 @@
     "family": {
       "base": {
         "comment": "Default font used within all Okta product user interfaces",
-        "value": "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif"
+        "value": "'Inter', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif"
```

Comment:
Do we provide `Inter` in our library or is it something apps are expected to have?

Reply:
We don't provide `Inter`, so this stack will immediately fall back to system fonts if the downstream teams haven't provided it / it's not available locally.